src/codegen/metadata/functions.py

```diff
@@ -12346,7 +12346,6 @@
                 'is_compound_type': True,
                 'max_length': 96,
                 'name': 'expStates',
-                'proto_only': True,
```

Comment:
FYI, `proto_only` field was previously added to the parameters that are of repeated variable argument type(`repeated_var_args`) in the metadata. But it appears that it was affecting `grpc-device`. Hence reverted back those changes in scrapigen.